src/cmake-tools.ts

```diff
@@ -167,6 +168,12 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
    */
   private _cmakeDriver: Promise<CMakeDriver|null> = Promise.resolve(null);
 
+  /**
+   * webview object
+   * handles everything regarding webview
+   */
+  private _webview: ConfigurationWebview | undefined;
```

Comment:
I recommend giving this a better name so that everyone knows this manages the Cmake GUI.